docs/graphql/manual/mutations/insert.rst

```diff
@@ -213,26 +213,32 @@ Insert an object and get a nested object in response
       }
     }
 
+Insert related data through relationships
```

Comment:
How about `Insert an object along with its related objects through relationships`. Seems to fit the naming convention of the other sections a little more snugly